evap/evaluation/views.py

```diff
@@ -198,29 +198,34 @@ def set_lang(request):
 
 def profile_edit(request):
     user = request.user
-    if user.is_editor:
-        form = DelegatesForm(request.POST or None, request.FILES or None, instance=user)
-
-        if form.is_valid():
-            form.save()
-
-            messages.success(request, _("Successfully updated your profile."))
-            return redirect("evaluation:profile_edit")
-
-        return render(
-            request,
-            "profile.html",
-            {
-                "user": user,
-                "form": form,
-                "delegate_of": user.represented_users.all(),
-                "cc_users": user.cc_users.all(),
-                "ccing_users": user.ccing_users.all(),
-            },
-        )
+    profile_form = ProfileForm(request.POST or None, request.FILES or None, instance=user)
+    form = DelegatesForm(request.POST or None, request.FILES or None, instance=user)
+
+    if request.method == "POST":
+        operation = request.POST.get("operation")
+        if operation not in ("profile", "delegates"):
+            raise SuspiciousOperation("Invalid POST operation")
+        if operation == "profile":
+            if profile_form.is_valid():
+                profile_form.save()
+                messages.success(request, _("Successfully updated your profile."))
+                return redirect("evaluation:profile_edit")
+
+        if user.is_editor and operation == "delegates":
+            if form.is_valid():
+                form.save()
+                messages.success(request, _("Successfully updated your profile."))
+                return redirect("evaluation:profile_edit")
 
     return render(
         request,
         "profile.html",
-        {"user": user},
+        {
+            "user": user,
+            "form": form,
+            "profile_form": profile_form,
+            "delegate_of": user.represented_users.all(),
+            "cc_users": user.cc_users.all(),
+            "ccing_users": user.ccing_users.all(),
+        },
```

Comment:
These now also get passed if the user is not an editor - it probably doesn't matter, but let's make this tight and leave them out for normal users?